test/src/test/java/org/corfudb/runtime/collections/CorfuStoreTest.java

```diff
@@ -264,6 +267,19 @@ public void checkMetadataTransactions() throws Exception {
         assertThatThrownBy(() ->
                 table.update(key1, EventInfo.getDefaultInstance(), wrongRevisionMetadata))
                 .isExactlyInstanceOf(RuntimeException.class);
+
```

Comment:
Can we have a test where the number of entries > batchsize and then verify that the number of partitions is correct and all entries are included in the entryStream api?

Reply:
this api is using the same parameters as checkpoint writer
I think these good suggestions but are tests we should add to the checkpointer writer probably not here right?